www/src/data/sidebars/contributing-links.yaml

```diff
@@ -1,87 +1,52 @@
+        - /contributing/how-to-file-an-issue/
+        - /contributing/how-to-make-a-reproducible-test-case/
+        - /contributing/how-to-open-a-pull-request/
+        - /contributing/how-to-label-an-issue/
+        - /contributing/triaging-github-issues/
+        - /contributing/managing-pull-requests/
+        - link: /contributing/docs-contributions/
           items:
-            - title: Docs Templates
-              link: /contributing/docs-templates/
-            - title: How to Write a Stub
-              link: /contributing/how-to-write-a-stub/
-            - title: Stub List
-              link: /contributing/stub-list/
-        - title: Blog & Website Contributions
-          link: /contributing/blog-and-website-contributions/
-        - title: Code Contributions
-          link: /contributing/code-contributions/
-        - title: Setting Up Your Local Dev Environment
-          link: /contributing/setting-up-your-local-dev-environment/
-        - title: Docs and Website Components
-          link: /contributing/docs-and-website-components/
-        - title: Community Contributions
-          link: /contributing/community-contributions/
+            - /contributing/docs-templates/
+            - /contributing/how-to-write-a-stub/
+            - /contributing/stub-list/
+        - /contributing/blog-and-website-contributions/
+        - /contributing/code-contributions/
+        - /contributing/setting-up-your-local-dev-environment/
+        - /contributing/docs-and-website-components/
+        - link: /contributing/community-contributions/
           items:
-            - title: Submit to Site Showcase
-              link: /contributing/site-showcase-submissions/
-            - title: Submit to Creator Showcase
-              link: /contributing/submit-to-creator-showcase/
-            - title: Submit to Starter Library
-              link: /contributing/submit-to-starter-library/
-            - title: Submit to Plugin Library
-              link: /contributing/submit-to-plugin-library/
+            - /contributing/site-showcase-submissions/
+            - /contributing/submit-to-creator-showcase/
+            - /contributing/submit-to-starter-library/
+            - /contributing/submit-to-plugin-library/
     - title: Gatsby Brand Guidelines
       items:
-        - title: Color
-          link: /guidelines/color/
-        - title: Logo
-          link: /guidelines/logo/
-        - title: Typography
-          link: /guidelines/typography/
-        - title: Design Tokens
-          link: /guidelines/design-tokens/
-    - title: RFC Process
-      link: /contributing/rfc-process/
-    - title: Gatsby's Governance Model*
-      link: /contributing/gatsby-governance-model/
+        - link: /guidelines/color/
+          title: Color
+        - link: /guidelines/logo/
+          title: Logo
+        - link: /guidelines/typography/
+          title: Typography
+        - link: /guidelines/design-tokens/
+          title: Design Tokens
```

Comment:
Should those titles be removed from here as well?

other `items` lists contain only links instead of object with link and title)

Reply:
Ah, ok I followed code and seems like that's ok